### PR TITLE
User id to id

### LINKED ISSUE #589: User_id to user.id
i prose changing for v2 the user_id field in the user table to id.
This is more logic in fact for database normalisation and.
Also this field is sometimes user_id in code and sometimes id. this should make it more logic.

### PR DESCRIPTION
**Pull request by stijnhau**
Changed the user_id field to id.
Fixes #589

### CODE REVIEW

File: data/schema.sqlite.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CREATE TABLE user
 (
-    user_id       INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    user          INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
```
Comment (Danielss89) [inline data/schema.sqlite.sql]: should be `id`

File: data/schema.sql [hunk 2]
```diff
@@ -1,6 +1,6 @@
 CREATE TABLE user
 (
-    user_id       INTEGER PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    user          INTEGER PRIMARY KEY AUTO_INCREMENT NOT NULL,
```
Comment (Danielss89) [inline data/schema.sql]: should be `id`

File: data/schema.pgsql.sql [hunk 3]
```diff
@@ -1,13 +1,13 @@
 CREATE TABLE public.user
 (                                                                                                                                                                                                                                                                              
-  user_id		serial NOT NULL,
+    user		serial NOT NULL,
```
Comment (Danielss89) [inline data/schema.pgsql.sql]: should be `id`

File: data/schema.mysql.sql [hunk 4]
```diff
@@ -1,6 +1,6 @@
 CREATE TABLE `user`
 (
-    `user_id`       INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `user`          INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
```
Comment (Danielss89) [inline data/schema.mysql.sql]: should be `id`

File: data/schema.ibmdb2.sql [hunk 5]
```diff
@@ -1,6 +1,6 @@
 CREATE TABLE user
 (
-    user_id       INTEGER PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    use           INTEGER PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
```
Comment (Danielss89) [inline data/schema.ibmdb2.sql]: should be `id`

**Comment by stijnhau** [comment]
Fixed the mistake in those sql filles.

File: data/schema.sqlite.sql (hunk 1 above)
Comment (Danielss89) [inline data/schema.sqlite.sql]: Still missing this

**Comment by stijnhau** [comment]
also corrected in In data/schema.sqlite.sql:

**Comment by Danielss89** [comment]
:+1:

**Comment by stijnhau** [comment]
https://github.com/ZF-Commons/ZfcUserDoctrineORM/pull/95/files
For the doctrine versions.

**Comment by stijnhau** [comment]
Any progress on this?

**Comment by Danielss89** [comment]
This is for 2.0 which is still in the future, so i'll keep them as PR's until i'm ready to do more about it.

**Comment by stijnhau** [comment]
Totaly fine you dont work hard on it now.
But i suggest merging already because of someone else does a PR it van create merge conflict.
And those can be a bitch

**Comment by stijnhau** [comment]
Any progress on this merge?

**Comment by stijnhau** [comment]
Maybe the PR can be merged in?

**Pull request merged**